README.md

```diff
@@ -144,7 +188,23 @@ Your query here
 8. Show the uuid of the employee, first_name and lastname combined, email, job_title, the name of the office they belong to, the name of the country, the name of the state and the name of the boss (boss_name)
 
 ```
-Your query here
+ SELECT em.uuid,
```

Comment:
This query should take into consideration all employees, even those that don't belong to an office or don't have a boss (supervisor).